### DISCUSSION TITLE
Zero length transcriptions

### DISCUSSION THREAD

**Discussion by PhiTiet** [openai/whisper#2249] [Q&A]
I'm using the transcriptions API and I am noticing that in my responses I get words which have effectively 0 duration.
I'm using these parameters in the request:  
```
POST -> https://api.openai.com/v1/audio/transcription 
responseformat = verbose_json
timestamp_granularities[] = word
model = whisper-1
```

For the most part, the transcription is accurate, but sometimes it will have a word where the start and end are seperated by less than a millisecond (which is also inaccurate when I listen to the mp3 which is also generated by openai)

Has anyone else had trouble with this/what can I do?

Example (I'm using java), see the word stairs.

<img width="755" alt="Screenshot 2024-06-27 at 10 02 05" src="https://github.com/openai/whisper/assets/47044950/413c8ac6-ec2a-4146-8a71-6533e6d8b680">

After looking at the average duration/letter I suspect the words with zero duration 'belong' to the word prior

**Comment by daskalou** [comment]
I can confirm we are seeing the same thing.

Regularly words are returned with 0 duration.

There's no consistency about whether they should belong to the prior word's timing or next word's.

This is in relatively short audio files, approximately 20-40 seconds long.
